media-video/blinkentools/blinkentools-2.9-r1.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Provides a set of commandline utilities related to Blinkenlights"
+HOMEPAGE="http://blinkenlights.net/project/developer-tools"
+SRC_URI="http://blinkenlights.de/dist/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="media-libs/blib
```

Comment:
not needed in `RDEPEND`? please check linking.

Reply:
checked and fixed.. it's needed..